exercises/harriet_brown/ex02_oo_basics/inc/Shape.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include <string>
```

Comment:
One of the most important things for us in C++ is to keep all unnecessary includes out of a file as these cause big increases in compile time as other files include headers. This is the "include what you use" rule. 

In particular `<iostream>` should never be used in a header as it's very heavy.